packages/fx-core/src/plugins/resource/frontend/v2/index.ts

```diff
@@ -48,8 +46,9 @@ export class FrontendPluginV2 implements ResourcePlugin {
   plugin!: FrontendPlugin;
 
   activate(projectSettings: ProjectSettings): boolean {
+    const activateInVS = projectSettings.programmingLanguage === TabLanguage.CSharp;
     const solutionSettings = projectSettings.solutionSettings as AzureSolutionSettings;
-    return this.plugin.activate(solutionSettings);
+    return this.plugin.activate(solutionSettings) || activateInVS;
```

Comment:
I'm not quite sure - do we need to make any change for v1?

Nit: The invocation of `this.plugin.activate` is meaningless  when `activateInVS` is true. Better to swap the position of `activateInVS` with the former `activate()`.